.github/workflows/run-k8s-tests-on-aks.yaml

```diff
@@ -27,78 +27,44 @@ jobs:
           - host_os: cbl-mariner
             vmm: clh
     runs-on: ubuntu-latest
+    env:
+      AZ_APPID: ${{ secrets.AZ_APPID }}
+      AZ_PASSWORD: ${{ secrets.AZ_PASSWORD }}
+      AZ_TENANT_ID: ${{ secrets.AZ_TENANT_ID }}
```

Comment:
I'm afraid those ones here would become plain text and easy to extract by having them as an env.
Wouldn't it be the case?  If not, why?

Reply:
This is safe per [[1]](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-secrets) and [[2]](https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow), but I did restrict the scope to only the login step to even further reduce the attack surface.